<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministrationPanel</class>
 <widget class="QMainWindow" name="AdministrationPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>679</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>808</width>
    <height>679</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>808</width>
    <height>679</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="user">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Hello,**%user%**</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnLogout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="administrationGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Aministration</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QPushButton" name="btnRemoveAdministration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QTableView" name="administrationView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnAddAdministration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnEditAdministration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="workersGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Workers</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="3">
        <widget class="QTableView" name="workersView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnRemoveWorker">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnAddWorker">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnEditWorker">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="professionsGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Professions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QPushButton" name="btnRemoveProfession">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnAddProfession">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnEditProfession">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QTreeView" name="professionsView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLogout</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddAdministration</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>addAdministration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEditAdministration</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>editAdministration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoveAdministration</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>removeAdministration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddWorker</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>addWorker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEditWorker</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>editWorker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoveWorker</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>removeWorker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddProfession</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>addProfession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEditProfession</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>editProfession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoveProfession</sender>
   <signal>clicked()</signal>
   <receiver>AdministrationPanel</receiver>
   <slot>removeProfession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doLogout()</slot>
  <slot>addAdministration()</slot>
  <slot>addWorker()</slot>
  <slot>addProfession()</slot>
  <slot>removeAdministration()</slot>
  <slot>removeWorker()</slot>
  <slot>removeProfession()</slot>
  <slot>editAdministration()</slot>
  <slot>editWorker()</slot>
  <slot>editProfession()</slot>
 </slots>
</ui>
